apps/src/studio/Item.js

```diff
@@ -38,14 +38,26 @@ var Item = function (options) {
   this.fadeTime = constants.ITEM_FADE_TIME;
 
   this.currentFrame_ = 0;
+  this.setAnimationRate(utils.valueOr(options.animationRate,
```

Comment:
might read a bit easier if newline happens before utils, but not sure what our standard is here.